src/message_handler.cc

```diff
@@ -121,9 +121,8 @@ void ReplyOnce::ReplyLocationLink(std::vector<LocationLink> &result) {
   if (g_config->client.linkSupport) {
     (*this)(result);
   } else {
-    std::vector<Location> result1;
-    for (auto &loc : result)
-      result1.emplace_back(std::move(loc));
+    std::vector<Location> result1{std::make_move_iterator(result.begin()),
```

Comment:
This is good.

Reply:
Use `(..)` instead of `{...}`. `{}` can be confusing because the constructor has an overload on initializer_list.